content/doc/developer/plugin-development/choosing-jenkins-baseline.adoc

```diff
@@ -33,7 +33,7 @@ These are updated monthly.
 When updating the core dependency, choose the link:/changelog-stable/[newest LTS version] that doesn't exclude a majority of your existing users (by requiring a newer Jenkins than they have).
 
 NOTE: If you are packaging a pure API library (one that does not depend on Jenkins APIs) then you should ignore newer Jenkins versions and pick an older LTS.
-Something around 1 year old that does not have too many detached plugins makes a good choice and 2.361.2 would be a reasonable candidate.
+Something around 1 year old that does not have too many detached plugins makes a good choice and 2.361.4 would be a reasonable candidate.
```

Comment:
Thanks for this change.  I think that this recommendation makes sense for consumers to depend on the last version of the older line rather than depending on an interim version of the older line.